app/views/spree/admin/shared/_order_tabs.html.erb

```diff
@@ -4,88 +4,15 @@
 <% end %>
 
 <% content_for :page_tabs do %>
-  <% if ((can? :update, @order) && (@order.shipments.size.zero? || @order.shipments.shipped.size.zero?)) %>
-    <li class="nav-item" data-hook='admin_order_tabs_cart_details'>
```

Comment:
I have not migrated these `data-hook` attributes as I couldn't find code that uses them. Are they needed? 🤔

Reply:
They're used by deface, if possible we should migrate them too. Some legacy extensions may depend on it

Reply:
We can deprecate them and get rid of them in some later versions